CODEOWNERS

```
@@ -60,6 +60,7 @@ homeassistant/components/axis/* @Kane610
 homeassistant/components/azure_devops/* @timmo001
 homeassistant/components/azure_event_hub/* @eavanvalkenburg
 homeassistant/components/azure_service_bus/* @hfurubotten
+homeassistant/components/barry/* @yash @ThomasBLarsen
```

You are @YashYKhatri ?

```suggestion
homeassistant/components/barry/* @YashYKhatri @ThomasBLarsen
```

Also does @ThomasBLarsen consent to also being a code owner?

---

Yes, My github name is @YashYKhatri and @ThomasBLarsen is also the code owner.